### PR TITLE
1111

### PR DESCRIPTION
**Pull request by rda-odoo**
eeeeeeeeeeeeeeeeeeeeeeeeeeeee

### CODE REVIEW

**Comment by rda-odoo** [comment]
pc1

File: __manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Cancel Journal Entries',
+    'version': '1.1',
+    'category': 'Accounting',
+    'description': """
+Allows canceling accounting entries.
+====================================
+
```
Comment (rda-odoo) [inline __manifest__.py:11]: hhhc2

File: __manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Cancel Journal Entries',
```
Comment (fka-odoo) [inline __manifest__.py:5]: wrong name define

File: __manifest__.py (hunk 1 above)
Comment (fka-odoo) [reply __manifest__.py:11]: sfsdfsdfsdaf

File: __manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Allows canceling accounting entries.
+====================================
+
+This module adds 'Allow Canceling Entries' field on form view of account journal.
+If set to true it allows user to cancel entries & invoices.
+    """,
+    'website': 'https://www.odoo.com/page/accounting',
+    'depends': ['account'],
+    'data': ['views/account_views.xml'],
+    'demo': [],
```
Comment (fka-odoo) [inline __manifest__.py:18]: remove demo

File: __manifest__.py (hunk 1 above)
Comment (fka-odoo) [reply __manifest__.py:11]: asdasdasdasd

Comment (fka-odoo) [inline __manifest__.py]: sddfsdfafsdf

**Pull request closed without merging**